cpp/src/io/parquet/page_enc.cu

```diff
@@ -1444,18 +1444,18 @@ __device__ std::pair<const void*, uint32_t> truncate_utf8(device_span<uint8_t co
   }
 
   // couldn't truncate, return original value
-  return {str.data(), str.size_bytes()};
+  return {span.data(), span.size_bytes()};
```

Comment:
Why not return the span here? It was passed by value and can be moved by the compiler instead of making a new span.

Reply:
it's still returning std::pair though

Reply:
That it is...